audit_exceptions/provided_by_macos_depends_on_allowlist.json

```diff
@@ -5,5 +5,6 @@
   "llvm",
   "openblas",
   "openssl@1.1",
-  "openssl@3"
+  "openssl@3",
+  "sqlite"
```

Comment:
I don't think we want to add this here -- system `sqlite` is good enough pretty often.

Reply:
Can you try a run with `uses_from_macos "sqlite"`?

Reply:
I can give a try, although the reason I had tried to force brewed `sqlite` here was to match what was done with other dependents of `libspatialite`, including `gdal` and `spatialite-gui` (which will depend on `librasterlite2` once it is updated).  My concern is that we may not easily be able to tell if there is a problem mixing the system `sqlite` and brewed `sqlite`.  

I will say that the issue referenced in `libspatialite` for why it is using brewed `sqlite` and not the system one is from 2014, and it may no longer be relevant.  We could consider trying to migrate `libspatialite` and all its dependents to `uses_from_macos "sqlite"` as that would avoid any concerns about mixing versions, but I think that would be better handled in its own PR, or potentially in the `proj@7` migration PR that originally motivated the addition of this formula.

Reply:
Yes, it's safer to avoid mixing library versions if we can help it.

Reply:
In that case I think this is ready to merge as the audit failures we do have are expected.